### PR TITLE
Reorganize columns in the topology dashboard

### PR DESCRIPTION
**Pull request by YianZong**
# Description
1. Reorganize columns in the topology table:
       -  The order of columns would be :
           Namespace > PVC > PV > StorageClass > Size > Status > Created > Storage System > Protocol > Pool > Volume
       -  "not expand" the CSI Driver
2. Change the external IP into internal one for security check
3. Update CodeOwner
4. Update action.yaml for image scanning check

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
|https://github.com/dell/csm/issues/401 |

# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [x] I have verified that new and existing unit tests pass locally with my changes
- [x] I have not allowed coverage numbers to degenerate
- [x] I have maintained at least 90% code coverage
- [x] I have inspected the Grafana dashboards to verify the data is displayed properly
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have maintained backward compatibility

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [x] UT
image

- [x] Manual
image

# Manual inspection of the GUI
I have verified that the dashboards show the data properly while generating I/O and storage resources

- [x] Yes
- [ ] No

### CODE REVIEW

**Comment by YianZong** [comment]
run e2e test

**Comment by YianZong** [comment]
run e2e test

**Comment by YianZong** [comment]
e2e pass

**Comment by YianZong** [comment]
Hi Code Owners,

Please kindly help review this PR.
The e2e test job is successful: https://osj-sio-03-prd.cec.lab.emc.com/view/CSI-CSM-Consolidation/job/Ecosystems_Novus/job/Data-Services-Broker/job/csm-metrics-powerstore-pr-based-e2e/46/

**Review by baoy1 — APPROVED** [review]

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -17,6 +17,11 @@
 # Shayna Finocchiaro (shaynafinocchiaro)
 # Logan Jones (lj-software)
```
Comment (shaynafinocchiaro) [inline .github/CODEOWNERS:18]: Please remove Logan and Michael from the code owners list. Thank you!

Comment (YianZong) [reply .github/CODEOWNERS]: Addressed

**Comment by YianZong** [comment]
run e2e test

**Comment by YianZong** [comment]
Updated the code owner file and the e2e is successful:
https://osj-sio-03-prd.cec.lab.emc.com/view/CSI-CSM-Consolidation/job/Ecosystems_Novus/job/Data-Services-Broker/job/karavi-topo-pr-based-e2e/56/

**Comment by YianZong** [comment]
e2e pass

**Review by baoy1 — APPROVED** [review]

**Review by P-Cao — APPROVED** [review]

**Review by forrestxia — APPROVED** [review]

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -17,6 +17,11 @@
...
 # Sharmila Ramamoorthy (sharmilarama)
+# Forrest Xia (forrestxia)
+# Yian Zong (YianZong)
+# Yiming Bao (baoy1)
+# Tao He (taohe1012)
+# Peter Cao (P-Cao)
 
 # for all files:
-* @medegw01 @gallacher @tdawe @alikdell @atye @hoppea2 @coulof @shaynafinocchiaro @lj-software @sharmilarama
+* @gallacher @tdawe @alikdell @atye @hoppea2 @coulof @shaynafinocchiaro @sharmilarama @forrestxia @YianZong @baoy1 @taohe1012 @P-Cao
```
[the remark this answers is not in the document]
Comment (YianZong) [reply .github/CODEOWNERS:25]: @forrestxia already removed

**Comment by YianZong** [comment]
Hi Code Owners,
Please kindly review the PR. Thanks!

**Review by shaynafinocchiaro — APPROVED** [review]

**Review by taohe1012 — APPROVED** [review]

**Review by sharmilarama — APPROVED** [review]

**Pull request merged**